mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -1580,6 +1580,24 @@ func ResourceContainerCluster() *schema.Resource {
 				},
 			},
 
+			"additional_pod_ranges_config": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
```

Comment:
The API doc mentions that the field additionalPodRangesConfig is output only, which means that this field should not be provided by the users and not updatable. But based on the tests, it looks like this field can be provided by the user and updatable. Do you know if the API doc is not accurate?  https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1beta1/projects.locations.clusters#ipallocationpolicy

Reply:
[This documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-pod-cidr#assign_additional_secondary_ranges_to_the_cluster) explains how to update that field via gcloud, so the intention is to allow users to update the field. 

While the additionalPodRangesConfig field itself is output_only, there are fields of the [ClusterUpdate message](https://cloud.google.com/kubernetes-engine/docs/reference/rest/v1/ClusterUpdate) that are available to modify additionalPodRangesConfig which are not output_only.